src/reformat_libyuv.c

```diff
@@ -1,6 +1,7 @@
 // Copyright 2019 Joe Drago. All rights reserved.
 // SPDX-License-Identifier: BSD-2-Clause
 
+#include "avif/avif.h"
```

Comment:
It is not necessary to include both avif.h and internal.h. By design, internal.h includes avif.h so that internal files just need to include internal.h.

Reply:
There should be a way to annotate internal.h so that tools like IWYU know it can be used as a substitute for avif.h. I just took a quick look at the IWYU documentation. I think it's `IWYU pragma: export`:

https://github.com/include-what-you-use/include-what-you-use/blob/master/docs/IWYUPragmas.md#iwyu-pragma-export

Could you test that? Thanks.

Reply:
I wrote https://github.com/AOMediaCodec/libavif/pull/1323.